include/deal.II/cgal/surface_mesh.h

```diff
@@ -66,6 +78,50 @@ namespace CGALWrappers
     const typename dealii::Triangulation<dim, spacedim>::cell_iterator &cell,
     const dealii::Mapping<dim, spacedim> &                              mapping,
     CGAL::Surface_mesh<CGALPointType> &                                 mesh);
+
+  /**
+   * Performs corefinement and boolean operations on two deal.II cells and put
+   * the result in a CGAL::Surface_mesh object.
+   *
+   * Extensive examples can be found in the CGAL documentation at
+   * https://doc.cgal.org/latest/Polygon_mesh_processing/index.html#coref_coref_subsec
+   *
```

Comment:
I would put here some examples compute with this function, similar to the one you have in the comments.